src/geodedic.rs

```diff
@@ -203,12 +226,35 @@ impl Coordinate<Ecef> {
 
     /// Converts an Earth-Centered, Earth-Fixed coordinate into latitude, longitude, and altitude.
     ///
-    /// Note that this conversion is not trivial and needs to be approximated. This implementation
-    /// currentaly uses [Ferrari's solution][ferrari], but this may change in the future.
+    /// Note that this conversion is not trivial and needs to be approximated.
+    ///
+    /// The implementation currently only guarantees conversion to Wgs84 datums with altitude
+    /// between -10km and 50km from the surface of the Wgs84 ellipsoid, roughly corresponding
```

Comment:
```suggestion
    /// The implementation currently only guarantees conversion to WGS84 datums with altitude
    /// between -10km and 50km from the surface of the WGS84 ellipsoid, roughly corresponding
```
The spec is WGS84, the Rust type is `Wgs84`.